client/my-sites/domains/domain-management/dataviews/dataviews-fields/domain-status-field.tsx

```diff
@@ -20,8 +22,8 @@ const DomainStatusField = ( props: Props ) => {
 	const translate = useTranslate();
 
 	const domain = getFullDomain( props.domain );
-	if ( ! domain ) {
-		return <></>;
+	if ( ! domain || isLoadingSites ) {
+		return <LoadingPlaceholder />;
 	}
```

Comment:
Should we handle the case when the domain is unavailable and show the LoadingPlaceholder only when the flag isLoadingSites is true?

Something like:

```
if ( isLoadingSites ) {
  return <LoadingPlaceholder />;
} else if ( ! domain ) {
  return <></>;
}
```

Reply:
Unfortunately, we can't split the condition this way: domain is the result of another async loading process:
- `isLoadingSites` reflects the status of loading information about sites,
- `domain` is part of site domains' queries result, I expect it to be always present in the end.